MAUI/Themes/HowTo.md

```diff
@@ -0,0 +1,41 @@
+---
+layout: post
+title: How to customize theming in Syncfusion controls
+description: Learn here about how to switch between light and dark themes in Syncfusion Maui controls and more details.
+platform: maui
+control: General
+documentation: ug
+---
+
+# Switching between light theme and dark theme
+
+To switch to light theme from dark theme, use the following code snippet.
```

Comment:
To switch between light and dark themes in Syncfusion Maui controls, use the following code snippet.

Reply:
Changes addressed.